contracts/test/MockVBNB.sol

```diff
@@ -361,7 +358,7 @@ contract MockVBNB is VToken {
         accrualBlockNumber = currentBlockNumber;
         borrowIndex = borrowIndexNew;
         totalBorrows = totalBorrowsNew;
-        totalReserves = totalReservesNew;
+        totalReserves = availableReserves;
```

Comment:
I would say reserves have to be increased always with `totalReservesNew`, independently of having enough cash to cover it